test/ambiguous.jl

```diff
@@ -107,7 +107,7 @@ ambs = detect_ambiguities(Ambig5)
 @test length(ambs) == 2
 
 # Test that Core and Base are free of ambiguities
-@test isempty(detect_ambiguities(Core, Base; imported=true))
+@test (x->(isempty(x) || println(x)))(detect_ambiguities(Core, Base; imported=true))
```

Comment:
that's intentional. the test macro doesn't show the contents of the result if isempty returns false, it just shows false. this still triggers an error but prints the contents